examples/lfric/full_example_netcdf/README

```diff
@@ -0,0 +1,76 @@
+This directory contains a stand-alone LFRic code example.
```

Comment:
I think this file needs to be Markdown (*.md) for consistency. Plus, most people will read this from a browser and Markdown will show better.

Reply:
Done.